File: frontend/henn-laszlo/projects/invitation-data-access/src/lib/models/invitation-list-item.ts
export interface InvitationListItem {
  readonly id: string;
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly year: number | null;
  readonly altTextHu: string | null;
  readonly altTextEn: string | null;
  readonly displayOrder: number;
}

File: backend/src/Modules/Modules.Invitation.Application/Invitations/GetAll/InvitationListItem.cs
namespace Modules.Invitation.Application.Invitations.GetAll;

public sealed record InvitationListItem(
    Guid Id,
    string TitleHu,
    string? TitleEn,
    int? Year,
    string? AltTextHu,
    string? AltTextEn,
    int DisplayOrder);